pkg/reconciler/certificate/resources/cert_manager_certificate.go

```diff
@@ -135,6 +138,29 @@ func MakeCertManagerCertificate(cmConfig *config.CertManagerConfig, knCert *v1al
 
 	dnsNames = append(dnsNames, knCert.Spec.DNSNames...)
 
+	var issuerRef cmeta.ObjectReference
+	if knCert.Labels[networking.VisibilityLabelKey] == VisibilityClusterLocal {
+		if cmConfig.InternalIssuerRef == nil {
+			return nil, &apis.Condition{
+				Type:    IssuerNotSetCondition,
+				Status:  corev1.ConditionFalse,
+				Reason:  "internalIssuerRef not set",
+				Message: "error creating cert-manager certificate: internalIssuerRef was not set in config-certmanager",
+			}
+		}
+		issuerRef = *cmConfig.InternalIssuerRef
+	} else {
```

Comment:
Seems like `IssuerRef` isused only if `internalIssuerRef` is not used. 
1. Do we really need two config points? 
2. This is not reflected in the config.yaml comments